core/src/com/unciv/logic/civilization/CivilizationInfo.kt

```diff
@@ -200,6 +200,8 @@ class CivilizationInfo : IsPartOfGameInfoSerialization {
         exploredTiles.addAll(tiles)
     }
 
+    var neutralRoads = HashSet<Vector2>()
```

Comment:
Since this is serialized, this means there are now 2 sources of truth for who owns what - one in the tile and one in the civilizationInfo
To solve this and have one source if truth, this should be transient (not serialized) and built up on setTransients

Reply:
Done